tinkerbell/stack/templates/hook.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if .Values.stack.hook.enabled }}
+{{- $hookDownloadJobEnabled := eq (include "stack.hookDownloadJobEnabled" .) "true" -}}
```

Comment:
is this comparison redundant?

true == true is true
false == true is false

couldn't we just use `include "stack.hookDownloadJobEnabled"` directly?

Reply:
sure